products/appengine/terraform.yaml

```
@@ -87,6 +87,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         update_mask_fields:
           - "ssl_settings.certificate_id"
           - "ssl_settings.ssl_management_type"
+  ApplicationUrlDispatchRules: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{project}}"
+    import_format: ["{{project}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "app_engine_application_url_dispatch_rule"
```

```suggestion
        name: "app_engine_application_url_dispatch_rules_basic_example"
```

+ update the filename of the example

---

Oops, that + got turned into a * since it was at the start of the line. Can you also update the filename of `templates/terraform/examples/app_engine_application_url_dispatch_rule.tf.erb` to match the example name after this change?

---

I realized that, updated the file name. I was getting compilation error.